website/content/docs/k8s/connect/cluster-peering/usage/create-sameness-groups.mdx

```diff
@@ -33,11 +29,11 @@ To create and use sameness groups in your network, complete the following steps:
 
 Before creating a sameness group, take the following actions to prepare your network:
 
-#### Check namespace and service naming conventions
+#### Check Consul namespaces and service naming conventions
 
-Sameness groups are defined at the partition level. As a result, Consul assumes that namespaces and services in a sameness group that have the same name are identical. This behavior occurs even when two different partitions in the group contain functionally different services that share a name and namespace. This situation would lead to errors, as requests will be sent to the incorrect service.
+Sameness groups are defined at the partition level. Consul assumes these partitions have identical configurations, including identical service names and identical Consul namespaces. This behavior occurs even when two partitions in the group contain functionally different services that share a common name and namespace. For example, if distinct services both named `api` were registered to two members of a sameness group, it could lead to errors because requests may be sent to the incorrect service.
```

Comment:
```suggestion
Sameness groups are defined at the partition level. Consul assumes all partitions in the group have identical configurations, including identical service names and identical Consul namespaces. This behavior occurs even when two partitions in the group contain functionally different services that share a common name and namespace. For example, if distinct services both named `api` were registered to different members of a sameness group, it could lead to errors because requests may be sent to the incorrect service.
```
For consistency with the VM version